frontend/src/App.tsx

```diff
@@ -10,14 +10,15 @@ import localStorageService from './services/LocalStorageService';
 import { useAppDispatch, useAppSelector } from './redux/hooks';
 import { getAccount, selectAccount, noTokenReducer } from './features/account/AccountSlice';
 import { loginWithRefreshToken } from './services/AccountAPI';
+import ViewPay from './components/Employees/ViewPay';
```

Comment:
Any reason why these two don't use the lazy import?

Reply:
These were auto-imported, I'll change them